nncf/common/factory.py

```diff
@@ -35,7 +35,7 @@ def create(model: TModel) -> NNCFGraph:
             from nncf.onnx.graph.nncf_graph_builder import GraphConverter
 
             return GraphConverter.create_nncf_graph(model)
-        if model_backend == BackendType.OPENVINO:
+        if model_backend in [BackendType.OPENVINO, BackendType.OPTIMUM]:
```

Comment:
I didn't get why you need `BackendType.OPTIMUM`?

Reply:
This is redundant code from my previous experiments, please ignore